sharktank/conftest.py

```diff
@@ -256,3 +256,23 @@ def get_iree_flags(request: FixtureRequest):
     model_path["iree_hal_target_backends"] = set_fixture_from_cli_option(
         request, "--iree-hal-target-backends", "iree_hal_target_backends"
     )
+
+
+def pytest_html_results_table_header(cells):
```

Comment:
Do you use these functions anywhere?
Nit: Maybe a small comment to what the functions are doing.

Reply:
Added a comment. Yeah, these functions allow us to customize the html reports and add the XFail Reason column in the https://nod-ai.github.io/SHARK-Platform.